db/seeds.rb

```diff
@@ -45,6 +51,8 @@
   organisation_id: organisation.id
 )
 
+Ip.create(address: '31.242.140.103', location: location_2)
```

Comment:
With this new session and IP, I would do this more dynamically, so it runs no risk of being a duplicate, and makes a bit more sense to the next developer. You could give every IP in location_1 a session like this:

```
location_1.ips.each_with_index do |ip, index|
  Session.create(start: (Time.now - index.day).to_s,
    success: index.even? ? true : false,
    username: "garry",
    siteIP: ip.address
  )
end
```
That way one location has logs and another doesn't. Also all the logs will be different

Reply:
You will obviously have to do this after all the Ips are created for location_1